aws/terraform/infra/README.md

```diff
@@ -0,0 +1,29 @@
+# Infra
+
+Stands up the base infrastructure required to deploy a Metaflow stack.
+
+Mostly stands up and configures the VPC.
+
+## AWS Resources
+
+### IAM
+
+IAM policy to allow access to the above S3 bucket and secret.
+
+### KMS
+
+KMS Key to encrypt/decrypt AWS Secrets Manager Secret.
+
+### S3
+
+S3 bucket for flow data input / output. This is not the Metaflow controlled S3 bucket but rather an external bucket 
```

Comment:
What is the rationale for a separate S3 bucket here?

Reply:
Saw the comment addressing this - https://github.com/Netflix/metaflow-tools/pull/26/files#r628554042. Would it make sense to allow the capability to add additional s3 buckets that AWS Batch has permissions to read from as a separate terraform substack? That way, users can pick and choose how many buckets they need to create - I am expecting most users will be okay using the same bucket for Metaflow and other data needs. At Netflix, we have everything in a single bucket.

Reply:
Yup! That's exactly the comment I was going to reference or copy and paste here haha

I think that functionality would be welcomed by most, but in my opinion I think we should target that in a separate follow up PR. If you're okay with that, I'm happy to create a ticket on this repository pointing to this comment here, the comment linked above and the rationale. 

I believe you're correct with most people probably being comfortable with using the same bucket. We specifically avoided it to prevent any possible manual mistakes that could cause us to interfere with data "owned" by Metaflow.